views/models/UserConfigurationsModel/index.js

```diff
@@ -0,0 +1,71 @@
+import { apiRoutes } from '../../routes/routes'
+import { BASE_REQUEST_OPTIONS } from '../../../constants'
+
+const UserConfigurationsModel = {
+  all: async (userId) => {
+    try {
+      const response = await fetch(
+        apiRoutes.configurations.userConfigurations(userId),
+        {
+          ...BASE_REQUEST_OPTIONS,
+        }
+      )
+      if (response.status !== 200) throw new Error(response.error)
+
+      return response.json()
+    } catch (error) {
+      throw new Error(error)
+    }
+  },
+  create: async (userId, configAttributes) => {
+    try {
+      const response = await fetch(
+        apiRoutes.configurations.userConfigurations(userId),
+        {
+          ...BASE_REQUEST_OPTIONS,
+          method: 'POST',
+          body: JSON.stringify(configAttributes),
+        }
+      )
+      const status = response.status
+
+      return { status, ...response.json() }
```

Comment:
why are we destructuring the json response here but putting it in a `data` key in the `update` method?

Reply:
makes sense i wanted to use the status from the response, but we lose it after we unmarshall the response.json 
now i'm using the key to assert it's existence
[d31c716](https://github.com/AMP-SCZ/dpdash/pull/310/commits/d31c716b584e340b90068954fc940c49761e52fe)